book/src/SUMMARY.md

```diff
@@ -38,6 +38,8 @@
   - [dbt](./integrations/dbt.md)
   - [Jupyter](./integrations/jupyter.md)
   - [Prefect](./integrations/prefect.md)
+  - [Python](./integrations/python.md)
+  - [Rust](./integrations/rust.md)
```

Comment:
These are listed on the website under `Bindings`, though it's a fuzzy categorization. WDYT?

Reply:
Oh, yes that makes sense to create a Bindings section I think. I'm also wondering why I didnt include the Javascript and Java ones.

New section called Bindings, with new additions Java and Javascript ?

Reply:
Would be perfect @charlie-sanders !